### PR TITLE
add porn-cli.profile

### PR DESCRIPTION
**Pull request by pirate486743186**
Python script for watching porn in the terminal.
a clone of mov-cli by the same devs. It's just a redirect to mov-cli
https://github.com/mov-cli/porn-cli

### CODE REVIEW

File: etc/profile-m-z/porn-cli.profile [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+# Description: Python script for watching porn via the terminal
+# This file is overwritten after every install/update
+quiet
+# Persistent local customizations
+include porn-cli.local
+# Persistent global definitions
+# added by included profile
+#include globals.local
+
+ignore include mov-cli.local
```
Comment (rusty-snake) [inline etc/profile-m-z/porn-cli.profile:11]: Do we habe this semantic anywhere else?

**Review by glitsj16 — APPROVED** [review]
LGTM

**Comment by kmk3** [comment]
Depends on #5710.

**Comment by netblue30** [comment]
all in, thanks!

**Pull request merged**